index.html

```diff
@@ -1,41 +1,41 @@
 ---
 title: Quantum Toolbox in Python
+layout: homepage
 ---
-{% include header.html %}
-{% include navbar.html %}
 
-<div class="row">
-<div class="col-md-9">
-<div class="row">
+{% include hero.html %}
 
-<div class="col-md-12">
-<div class="alert alert-success">
-<p id="cite"></p>
-</div>
-</div>
+<div class="row">
+    <div class="col-md-9">
+        <div class="row">
+            <div class="col-md-12">
+                <div class="banner" style="margin-top: 0;">
+                    <p id="cite"></p>
+                </div>
+            </div>
+        </div>
 
 <!-- <div class="col-md-12">
```

Comment:
Let's just delete this.

Reply:
You mean the commented part, right?
On that note, should we do the same for the commented out "jobs" banner?